packages/mobile/src/tokens/saga.test.ts

```diff
@@ -107,6 +109,53 @@ describe(fetchTokenBalancesSaga, () => {
       error: 'Error message',
     })
   })
+
+  describe('account event', () => {
+    let returnValueFirstTimeOnly: (value: BigNumber) => EffectProvider<any>
+
+    beforeEach(() => {
+      returnValueFirstTimeOnly = (value: BigNumber) => {
```

Comment:
this looks strange, i know. what i want to do is mock the outcome of the totalTokenBalanceSelector differently the first and second time it is called. this is surprisingly not out of the box, and the closest thing i could get to is [this](https://github.com/jfairbank/redux-saga-test-plan/blob/master/docs/integration-testing/mocking/static-providers.md#static-providers-with-dynamic-values).